src/ecs/explorer/ecsContainerNode.ts

```diff
@@ -43,4 +41,18 @@ export class EcsContainerNode extends AWSTreeNodeBase {
     public describeTasks(tasks: string[]) {
         return this.ecs.describeTasks(this.clusterArn, tasks)
     }
+
+    private async hasRunningTasks(): Promise<boolean> {
+        const tasks = await this.ecs.listTasks(this.clusterArn, this.serviceName, TASK_STATUS_RUNNING)
+        return tasks.length > 0
+    }
+
+    public async refresh(commands: Commands = Commands.vscode()): Promise<void> {
+        await this.updateRunningTasks()
+        return commands.execute('aws.refreshAwsExplorerNode', this)
```

Comment:
If this is needed, it needs to be `this.refresh()`, for the same reason as we did so in https://github.com/aws/aws-toolkit-vscode/pull/2359

Reply:
No, this is not needed at this time. I was thinking it might be useful in the future, but happy to leave it out.